### ISSUE TITLE
standard choices for sec. in rate

### ISSUE THREAD

**Issue by huss** [OpenEnergyDashboard/OED#1216]
### Is your feature request related to a problem? Please describe.

On unit create/edit, there is a box to enter the Sec. in Rate as a number. It would be nice if the admin had a list of standard values and also could enter a custom value if desired.

### Describe the solution you'd like

The list of standard values would be: Per Second (1), Per Minute (60) and Per Hour (3600).

The UI to allow this list and a custom value is unclear. OED has places where there are fixed value menus using an enum but not then allowing a custom value.

### Describe alternatives you've considered

Leave as is if this is difficult to do.

### Additional context

None.

**Comment by huss** [comment]
PR #1249 allows the bar width to be entered via check boxes and text. OED should probably be consistent in the interface so modeling this after that would be good.

Update: The check boxes are gone and it was decided to use a drop down menu. That is the wave of the OED future :-)

**Comment by AchsahJojo** [comment]
Me and my Team are working on this !

**Comment by SalvadorN323** [comment]
Hello! Is #1216 available to be worked on? If so, my team and I can look into it!

**Comment by AchsahJojo** [comment]
Hi ! me and my team were working on this issue over the summer and we are
close to being done but we had some issues that only the project maintainer
can solve and so we are waiting on him. Once that is done we will be able
to do a pull request.

**Comment by huss** [comment]
> Hello! Is #1216 available to be worked on? If so, my team and I can look into it!

@SalvadorN323 Let me confirm that this issue is still assigned (as shown above) and is not available to work on. If you want help in finding another issue then let me know.
